Resources/Prototypes/Catalog/Cargo/cargo_vending.yml

```diff
@@ -33,7 +33,7 @@
     sprite: Objects/Specific/Service/vending_machine_restock.rsi
     state: base
   product: CrateVendingMachineRestockClothesFilled
-  cost: 7250 #DeltaV
+  cost: 8050 #Einstein Engines
```

Comment:
Comments aren't needed on EE

Reply:
> Comments aren't needed on EE afaik

until there is documentation I will simply play it safe for now.

Reply:
```suggestion
  cost: 8050
```

Reply:
so is this the standard @DEATHB4DEFEAT ?

Reply:
We're EE